app/models/resident/properties/PropertyTotalGainModel.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+  * Copyright 2016 HM Revenue & Customs
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIED OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package models.resident.properties
+
+import common.Validation
+import models.resident.shares.TotalGainModel
+import play.api.mvc.QueryStringBindable
+
+case class PropertyTotalGainModel (totalGainModel: TotalGainModel, improvements: Double)
+
+object PropertyTotalGainModel {
+
+  implicit def propertyTotalGainBinder(implicit totalGainBinder: QueryStringBindable[TotalGainModel],
+     doubleBinder: QueryStringBindable[Double]) : QueryStringBindable[PropertyTotalGainModel] =
+      new QueryStringBindable[PropertyTotalGainModel] {
+        override def bind(key: String, params: Map[String, Seq[String]]): Option[Either[String, PropertyTotalGainModel]] = {
```

Comment:
Requires wrapping if statement for missing values (see current examples after rebasing)